lib/oli/analytics/datashop/elements/event_descriptor.ex

```diff
@@ -44,26 +44,20 @@ defmodule Oli.Analytics.Datashop.Elements.EventDescriptor do
 
           case activity_type(part_attempt) do
             # for short answer questions, the input is the text the student entered in the field
-            "oli_short_answer" -> input
+            "oli_short_answer" ->
```

Comment:
Platform code should not make any assumptions about the details of an activity's implementation.  We should provide a mechanism in the activity to specify what this datashop logic needs - rather than have this logic embed assumptions about the activity impl